### ISSUE TITLE
 Image got from imagestream quality is low. 

### ISSUE THREAD

**Issue by sejun2** [Apparence-io/camera_awesome#119]
## Steps to Reproduce

```
     SizedBox(
              height: 500,
              child: CameraAwesome(

                ///Image_stream function
                  imagesStreamBuilder: (imageStream) {
                    imageStream!.listen((data) async {
                      if (streamMutex) {
                        print('imageStream working...');
                        streamMutex = false;
                        //logic here...
                        //await writeImageFile(data);

                        //add image data to stream
                        previewDataStreamController.add(data);

                        await Future.delayed(const Duration(milliseconds: 500),
                                () {
                              streamMutex = true;
                            }); //await time - 700 milliseconds
                      }
                    });
                  },
                  photoSize: _size,
                  sensor: _sensor,
                  captureMode: _captureMode),
            ),

....

     StreamBuilder(
                        builder: (context, snapshot) {
                          var result = snapshot.data;
                         //print('snapshot.data : $result');

                          if (!snapshot.hasData || snapshot.hasError) {
                            return const Text('something\'s wrong');
                          } else {
                            return Expanded(
                              child: //Image.file(result as File),
                              Image.memory(
                                result as Uint8List,
                              ),
                            );
                          }
                        },
                        stream: previewDataStream),
                  ],
                )),
```

## Expected results

I expected origin quality of preview but it is not.

## Actual results

![Screenshot_1637549784](https://user-images.githubusercontent.com/33044667/142794452-0795345a-2fbd-4e78-b1a4-af9105eb8e7f.png)
The image under the camerapreview is abnormal.

## About your device

| Brand   | Model       | OS        |
| ------- | ----------- | --------- |
| *Apple* | *iPhone X*  | *13.6.1*  |

Android /emulator/ API 31

</details>

**Comment by ryuujo1573** [comment]
Is that about configs?  `photoSize: _size`

**Comment by g-apparence** [comment]
This should be fixed with the reworked version #150 

The new version will let you define the image stream definition and format.
